data/a-a.ts

```diff
@@ -0,0 +1,32 @@
+// TODO
+const TEST_CFR = {};
+
+export default {
```

Comment:
Do we have/need a schema for these experiment types? (so we know that they each have the necessary required fields, etc)

Most certainly not proper TypeScript interface syntax (and probably needs a sub "ExperimentTypePresets" interface), but possibly something adjacent to this:

```typescript
interface ExperimentType {
  name: string;
  description: string;
  presets: {
    max_bucket_count: number;
    branches: Array<Branch>;
    bucketing: BucketConfig;
  }
}
```